setup.py

```diff
@@ -12,7 +12,7 @@
     "Click~=6.0",
     "click-log==0.1.8",
     "distro~=1.3",
-    "elasticsearch5~=5.5",
+    "elasticsearch>=7.0",
```

Comment:
does this change make our previous version of ES backward-incompatible?

Reply:
If an app developer has an existing index in an older version of ES, we should recommend that they do a clean build to recreate those indices with ES7. Otherwise, all exposed ES functionality via MindMeld methods will continue to work the same way.